code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -221,12 +224,18 @@
 /obj/item/robot_module/proc/do_transform_delay()
 	var/mob/living/silicon/robot/R = loc
 	var/prev_lockcharge = R.lockcharge
-	sleep(1)
-	flick("[cyborg_base_icon]_transform", R)
+	if(animated_transform == TRUE)
+		flick("[cyborg_base_icon]_transform", R)
+	else
+		var/obj/effect/temp_visual/decoy/fading/fivesecond/ANM = new /obj/effect/temp_visual/decoy/fading/fivesecond(R.loc, R)
+		ANM.layer = layer - 0.01
+		new /obj/effect/temp_visual/small_smoke(R.loc)
+		R.alpha = 0
+		animate(R, alpha = 255, time = 50)
 	R.notransform = TRUE
 	R.SetLockdown(TRUE)
 	R.anchored = TRUE
-	sleep(1)
+	sleep(2)
```

Comment:
?

Reply:
Old code had two separate sleep procs both 1 second each, I just removed one and added another second here.